src/pages/PlanPage/subRoutes/CurrentOrgPlan/CurrentOrgPlan.tsx

```diff
@@ -35,6 +45,20 @@ function CurrentOrgPlan() {
       <InfoMessageStripeCallback />
       {accountDetails?.plan ? (
         <div className="flex flex-col gap-4 sm:mr-4 sm:flex-initial md:w-2/3 lg:w-3/4">
+          {!!planUpdatedNotification?.variant && (
+            <Alert variant={planUpdatedNotification.variant}>
+              {scheduleStart && scheduledPhase?.quantity && (
+                <Alert.Title>Plan successfully updated.</Alert.Title>
+              )}
+              {!!planUpdatedNotification?.variant && (
+                <Alert.Description>
+                  {scheduleStart && scheduledPhase?.quantity
+                    ? `The start date is ${scheduleStart} with a monthly subscription for ${scheduledPhase.quantity} seats.`
+                    : 'Plan successfully updated.'}
+                </Alert.Description>
+              )}
+            </Alert>
+          )}
```

Comment:
Can we swap out `&&` for ternaries instead, they provide a bit more protections over some ... interesting behaviour that can occur, you can [read more here](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx). I know that there are plenty of examples of the prior but we're trying to swap over to better protect ourselves moving forward.